contracts/factories/DelegatedManagerFactory.sol

```diff
@@ -373,6 +390,29 @@ contract DelegatedManagerFactory {
         });
     }
 
+    /**
+     * Initialize fee settings on DelegatedManager and transfer `owner` and `methodologist` roles.
+     *
+     * @param  _manager                 Instance of DelegatedManager
+     * @param  _owner                   Address that will be given the `owner` DelegatedManager's role
+     * @param  _methodologist           Address that will be given the `methodologist` DelegatedManager's role
+     * @param  _ownerFeeSplit           Percent of fees in precise units (10^16 = 1%) sent to owner, rest to methodologist
+     * @param  _ownerFeeRecipient       Address which receives owner's share of fees when they're distributed
+     */
+    function _setManagerState(
```

Comment:
Move manager state logic in `DelegatedManagerFactory.initialize` into an internal function